src/helm/benchmark/scenarios/harm_bench_scenario.py

```diff
@@ -0,0 +1,55 @@
+import os
+import pandas as pd
+from typing import List
+
+from helm.common.general import ensure_file_downloaded
+
+from .scenario import Scenario, Instance, Input, TEST_SPLIT, Reference, Output
+
+
+class HarmBenchScenario(Scenario):
+    """
+    HarmBench, a standardized evaluation framework for automated red teaming.
+    """
+
+    name = "harm_bench"
+    description = "HarmBench dataset - standard and contextual attacks"
+    tags = ["safety"]
+
+    def get_instances(self, output_path: str) -> List[Instance]:
+
+        data_path = os.path.join(output_path, "data")
+        os.makedirs(data_path, exist_ok=True)
+        url_prefix = "https://raw.githubusercontent.com/centerforaisafety/HarmBench/main/data/"
+        ensure_file_downloaded(
+            source_url=url_prefix + "behavior_datasets/harmbench_behaviors_text_all.csv",
+            target_path=os.path.join(data_path, "harmbench_behaviors_text_all.csv"),
```

Comment:
Make `os.path.join(data_path, "harmbench_behaviors_text_all.csv")` a named variable and reuse it below.